package.json

```diff
@@ -106,7 +106,7 @@
     "files:flip-to-nightly:windows": "./scripts/flip-files-to-nightly.ps1",
     "files:invalidate-bucket": "./scripts/invalidate-files-bucket.sh",
     "files:invalidate-bucket:nightly": "./scripts/invalidate-files-bucket.sh --nightly",
-    "postinstall": "./node_modules/.bin/electron-rebuild",
+    "postinstall": "yarn --cwd ./rust/kcl-language-server --modules-folder node_modules install && ./node_modules/.bin/electron-rebuild",
```

Comment:
What sorcery is this? It's neat

Reply:
https://docs.npmjs.com/cli/v8/using-npm/scripts#npm-install

NPM has lifecycles which is important because sometimes doing installs, builds etc... can trigger binaries to be built. There can be some oddities with packages that don't do something until the `npm` life cycle triggers it.

I think hes updated this to automatically update the nested library we have since `yarn tsc` points to it.

Reply:
This is needed for the kcl stuff jess moved recently. I asked her what do I do and she said "run this command after install". Well I knew npm has a special `postinstall` property, so I made use of that, here